### PR TITLE
Lesson 03 Part 02: Create Bash script

### PR DESCRIPTION
**Pull request by Ayrum1158**
Done lesson 3 part 2 task: Create Bash script

### CODE REVIEW

File: 01-Git_Bash_Make/Bash_o_script.script [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+if [ ! -d ~/tmp ]; then
+	mkdir ~/tmp
+fi
+
+touch ~/tmp/guessanumber
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Bash_o_script.script]: touch is using for file creation, not for folder

File: 01-Git_Bash_Make/Bash_o_script.script [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+if [ ! -d ~/tmp ]; then
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Bash_o_script.script]: /tmp/guesanumber is full path, not relative

File: 01-Git_Bash_Make/Bash_o_script.script [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Bash_o_script.script:1]: please define pathes as variable

File: 01-Git_Bash_Make/Bash_o_script.script (hunk 2 above)
Comment (Ayrum1158) [reply 01-Git_Bash_Make/Bash_o_script.script]: There is ~

File: 01-Git_Bash_Make/Bash_o_script.script [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+if [ ! -d ~/tmp ]; then
+	mkdir ~/tmp
+fi
+
+touch ~/tmp/guessanumber
+
+cat guessanumber.c > ~/tmp/guessanumber
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Bash_o_script.script]: - for copy, please use cp command
- you should copy all c/cpp files /tmp/guesanumber

**Pull request merged**